HealthcareSystem/PatientWebApp/wwwroot/js/patient_examinations.js

```diff
@@ -386,4 +328,102 @@ function cancelExamination(id) {
 
 function FillOutTheSurvey(examinationId) {
     window.location.href = '/html/filling_out_the_survey.html?id=' + examinationId;
+};
+
+function step(id) {
+
```

Comment:
Just curiosity, what is the advantage of this being one function instead of three? It behaves differently depending on the step anyway, and I don't see any common bits, why not just separate it?